templates/home/layout-home.html

```diff
@@ -36,7 +36,7 @@
 </header>
 <main property="mainContentOfPage" resource="#wb-main" typeof="WebPageElement">
 {{ content }}
-{% include main-footer/inc-footer.html %}
+{% include main-footer/inc-footer.html isFluid=true %}
```

Comment:
Feels like siFluid is counter intuitive and should be false when we want sections in container.

Reply:
Feel free to replace it with "addContainer" include parameter instead of "isFluid". If you do, please update all other reference.